### PR TITLE
Update requirements.md

### PR DESCRIPTION
**Pull request by MikeyMJCO**

### CODE REVIEW

**Comment by MikeyMJCO** [comment]
#205 #203

File: docs/requirements.md [hunk 1]
```diff
@@ -62,17 +62,24 @@ See our page on [choosing a CMS](/planning/cms.md) for more information about th
 
 To install these extensions, you will typically install a separate package within your server's package manager (e.g. `apt-get` on Ubuntu).
 
-* [PHP BCMath](https://www.php.net/bcmath) - required for many payment processors (including Stripe).
-* [PHP Curl](http://www.php.net/curl) - required for many payment processors, the extension manager, and the CiviCRM News dashlet
+#### Required for CiviCRM Core
+
+* [PHP BCMath](https://www.php.net/bcmath) - required for many payment processors.
```
Comment (eileenmcnaughton) [inline docs/requirements.md]: Actually no - required to calculate financial values - it really is used in the heart of civi as @mlutfy pointed out

Comment (MikeyMJCO) [reply docs/requirements.md]: Clarified.

File: docs/requirements.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ To install these extensions, you will typically install a separate package withi
 
 #### Required for CiviCRM Core
 
-* [PHP BCMath](https://www.php.net/bcmath) - required for many payment processors.
+* [PHP BCMath](https://www.php.net/bcmath) - required for calculating financial values in CiviCRM Core and many payment processors.
```
Comment (eileenmcnaughton) [inline docs/requirements.md]: actually - we don't even know if it IS used in many payment processors - the only mentioned one we know doesn't use it & I am not aware any that do!

Comment (MikeyMJCO) [reply docs/requirements.md]: Ok dropped bit about payment processors entirely.

**Comment by eileenmcnaughton** [comment]
OK - I think this is good now

**Pull request merged**